read_write_test.go

```diff
@@ -150,3 +150,107 @@ func TestDecode(t *testing.T) {
 	assert.Equal(t, "admin", dc.Contacts[1].Type, "contact type found")
 	assert.Equal(t, "some-password", dc.AuthInfo.Password, "auth info found")
 }
+
+func ExampleAddNamespace() {
+	// Construct the response with basic data.
+	diResponse := types.DomainInfoDataType{
+		InfoData: types.DomainInfoData{
+			Name: "example.se",
+			ROID: "DOMAIN_0000000000-SE",
+			Status: []types.DomainStatus{
+				{
+					DomainStatusType: types.DomainStatusOk,
+				},
+			},
+			Host: []string{
+				"ns1.example.se", "ns2.example.se",
+			},
+			ClientID: "Some Client",
+			CreateID: "Some Client",
+			UpdateID: "Some Client",
+		},
+	}
+
+	// Add extension data from extension iis-1.2.
+	diIISExtensionResponse := types.IISExtensionInfoDataType{
+		InfoData: types.IISExtensionInfoData{
+			State:        "active",
+			ClientDelete: false,
+		},
+	}
+
+	// ADd extension data from secDNS-1.1.
+	diDNSSECExtensionResponse := types.DNSSECExtensionInfoDataType{
+		InfoData: types.DNSSECOrKeyData{
+			DNSSECData: []types.DNSSEC{
+				{
+					KeyTag:     195550,
+					Algorithm:  3,
+					DigestType: 5,
+					Digest:     "FFAB0102FFAB0102FFAB0102FFAB0102FFAB0102FFAB0102FFAB0102FFAB0102",
+				},
+			},
+		},
+	}
+
+	response := types.Response{
+		Result: []types.Result{
+			{
+				Code:    EppOk.Code(),
+				Message: EppOk.Message(),
+			},
+		},
+		ResultData: diResponse,
+		Extension: struct {
+			types.IISExtensionInfoDataType
+			types.DNSSECExtensionInfoDataType
+		}{
+			diIISExtensionResponse,
+			diDNSSECExtensionResponse,
+		},
+	}
+
+	b, _ := Encode(response, ClientXMLAttributes())
+
+	fmt.Println(string(b))
+
+	// Output:
+	// <?xml version="1.0" encoding="UTF-8"?>
+	// <epp xmlns="urn:ietf:params:xml:ns:epp-1.0">
+	//   <response>
+	//     <result code="1000">
+	//       <msg>Command completed successfully</msg>
+	//     </result>
+	//     <resData>
+	//       <domain:infData xmlns:domain="urn:ietf:params:xml:ns:domain-1.0" xmlns="urn:ietf:params:xml:ns:domain-1.0">
```

Comment:
is `urn:ietf:params:xml:ns:domain-1.0` added twice?

Reply:
Yeah, see my comment [here](https://github.com/bombsimon/epp-go/pull/6#discussion_r270671116). Haven't figured out a way to fix this.